linode/objbucket/resource.go

```diff
@@ -59,7 +59,7 @@ func readResource(
 
 	regionOrCluster, label, err := DecodeBucketID(ctx, d.Id())
```

Comment:
We might want to add some logic to recover resources that have the corrupted `:bucketName` ID here

Reply:
Thanks for the suggestion, added.